vinca/templates/activate.sh.in

```diff
@@ -22,5 +22,9 @@ esac
 export ROS_ETC_DIR=$CONDA_PREFIX/etc/ros
 export AMENT_PREFIX_PATH=$CONDA_PREFIX
 
+if [ "$OSTYPE" = "darwin*" ]; then
+    export RMW_IMPLEMENTATION="rmw_cyclonedds_cpp"
+fi
```

Comment:
```suggestion
if [[ "$OSTYPE" == darwin* ]]; then
    export RMW_IMPLEMENTATION="rmw_cyclonedds_cpp"
fi
```

On bash, this works (See https://stackoverflow.com/questions/2172352/in-bash-how-can-i-check-if-a-string-begins-with-some-value). not sure on zsh